### ISSUE TITLE
`perl:podlators` version `v6.0.2` is not detected as the latest version

### ISSUE THREAD

**Issue by mtelka** [repology/repology-rules#907]
**Projects affected**

https://repology.org/project/perl:podlators

**Observed behavior**

The latest version `v6.0.2` listed at [MetaCPAN](https://metacpan.org/dist/podlators) is not detected as the latest one.  Only `6.0.2` is considered latest.

**Expected behavior**

Version `v6.0.2` should be accepted as the latest one too.

**Comment by AMDmi3** [comment]
Works as expected. Including `v` into the version is an error.

**Comment by AMDmi3** [comment]
> unfortunately, some people does so

Sure, and they should fix it, not us.

> And since MetaCPAN could be considered as an authoritative

It does not matter how authoritative it is if it provides garbage data. And I won't even call it authoritative given how many versions are outdated in then even not taking `v`'s into account.

**Comment by mtelka** [comment]
> > And since MetaCPAN could be considered as an authoritative
> 
> It does not matter how authoritative it is if it provides garbage data. And I won't even call it authoritative given how many versions are outdated in then even not taking `v`'s into account.

This is interesting.  I looked at it a bit and I think most of them are due to the `v`.  Some others needs merge (like `Alien-Base`).

**Comment by AMDmi3** [comment]
There's a lot of inconsistencies in Perl ecosystem, with versioning (`v`'s, or the fact that `1.2.3` == `1.002003`), naming and the way submodules are handled (may have or have no separate entries), and CPAN, MetaCPAN, and upstreams disagree on most of that.

**Comment by mtelka** [comment]
Yes, exactly.  The strange version numbers are total crap.  OTOH, after the long time spent there I learned that modules/submodules are not very important for packaging while distributions are.  I recently reimplemented our (OpenIndiana) automatic packaging tool to [work with distributions only](https://github.com/OpenIndiana/oi-userland/pull/19861).
